css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,22 @@
+button {
+  padding: 10px 25px;
+  font: 20px Arial, sans-serif;
+}
+
+button:hover {
+  background-color: black;
+  color: white;
+}
+
+table {
+  border: 2px solid white;
+}
+
+th {
+  background-color: black;
+  color: white;
+}
+
+.even {
```

Comment:
you want to use the nth-child pseudo class to complete this. Try refactoring using that!